cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -88,9 +88,17 @@ spec:
           requests:
             storage: {{ $root.Values.persistentVolume.storage }}
     {{- end }}
-
+  {{ if $root.Values.waitTime }}
+  minReadySeconds: {{ $root.Values.waitTime }}
```

Comment:
it is not clear on what the waitTime is for, may be we need to have it under updateStrategy or something?. That way in future we can even change the update to be non rolling if needed.

something like
```
updateStrategy:
    type: RollingUpdate
    delaySecond: 180
    masterPartition: 0
    tserverPartition: 3
```